src/Arcus.Messaging.Pumps.ServiceBus/Configuration/IAzureServiceBusQueueMessagePumpOptions.cs

```diff
@@ -47,11 +48,13 @@ public interface IAzureServiceBusQueueMessagePumpOptions
         /// <summary>
         /// Gets the options to control the correlation information upon the receiving of Azure Service Bus messages in the <see cref="AzureServiceBusMessagePump"/>.
         /// </summary>
+        [Obsolete("Will be moved to message routing options in the future")]
         AzureServiceBusCorrelationOptions Correlation { get; }
         
         /// <summary>
         /// Gets the consumer-configurable options to change the deserialization behavior.
         /// </summary>
+        [Obsolete("Will be moved to message routing options in the future")]
```

Comment:
Message routing options or the MessageRoutingOptions ?

Reply:
Well, it will probably be called `Routing` or `MessageRouting` to be consistent, but since it's not available yet in the interface (only in the implementation), I wouldn't want to fixate on the name in the deprecated message.